pytest.ini

```
@@ -2,7 +2,7 @@
 # -rsxX        - show skipped, failed, and passed tests
 # --tb=native  - print native traceback
 # -p no:django - disable the pytest-django plugin for Socorro tests
-addopts = -rsxX --tb=native -p no:django
+addopts = -rsxX --tb=native -p no:django -m 'not gcp'
 norecursedirs = .git docs config docker __pycache__
 testpaths = socorro/
```

pretty sure marker definitions need to be copied from webapp/pytest.ini as well:
```suggestion
addopts = -rsxX --tb=native -p no:django -m 'not gcp'
norecursedirs = .git docs config docker __pycache__
testpaths = socorro/

markers =
    aws: tests that require aws backends to be configured in the environment. this is the default.
    gcp: tests that require gcp backends to be configured in the environment. skipped unless explicitly requested.
```